packages/Tag/src/Tag.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import PropTypes from "prop-types";
+import * as sc from "./Tag.styles";
+import * as types from "./types";
+
+export default function Tag(props) {
+  const { children, onRemove, size, onClick, a11yText, avatar, isDisabled, ...moreProps } = props;
+
+  const handleRemove = e => {
+    onRemove();
+    e.stopPropagation();
+  };
+
+  const content = (
+    <>
+      {avatar}
```

Comment:
I think you should handle this the same as the delete button –conditionally render it and wrap in a `<sc>` component for the styling (padding).

Reply:
I'm not sure I see the benefits of this, the delete button should be conditionally rendered as there is a prop (callback) being passed and not an element that should be rendered, so slightly different. The styling that gets applied to avatar is only if it exists... so don't really see that it needs an extra layer of being a styled component...

Reply:
Fair enough.